doc/features/README.md

```diff
@@ -37,6 +37,7 @@
       + [Troubleshooting Endpoint Discovery](#troubleshooting-endpoint-discovery)
       + [Troubleshooting Pod Health Check](#troubleshooting-pod-health-check)
       + [Troubleshooting aci-containers-host](#troubleshooting-aci-containers-host)
+- [acikubectl trace](#acikubectl_trace)
```

Comment:
change to:
[Datapath tracing with acikubectl](#datapath-tracing-with-acikubectl)